mi/values_local.yaml

```diff
@@ -70,6 +78,21 @@ wso2:
         fileName: "client-truststore.jks"
         # -- Primary truststore password
         password: "wso2carbon"
+    icp:
```

Comment:
Let's add this at [values_full](https://github.com/wso2/helm-mi/pull/17/files#diff-bcc76be4bef7641b019320fd0866ae13f9344bdca77e5abdc9d1a413f542665bL213) as well.